app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_names = command.split()
+    if len(file_names) != 3 or file_names[0] != "cp":
+        return
```

Comment:
Consider providing feedback to the user when the command format is incorrect. Currently, the function silently returns without any indication of what went wrong.